virtcontainers/container_test.go

```diff
@@ -275,6 +275,57 @@ func TestContainerAddDriveDir(t *testing.T) {
 	}
 }
 
+func TestContainerRootfsPath(t *testing.T) {
+
+	testRawFile, loopDev, fakeRootfs, err := testSetupFakeRootfs(t)
+	defer cleanupFakeRootfsSetup(testRawFile, loopDev, fakeRootfs)
+	assert.Nil(t, err)
+
+	truecheckstoragedriver := checkStorageDriver
```

Comment:
@ganeshmaharaj do you still need this hack?

Reply:
@mcastelino yes. That particular functions check if the device is a devicemapper based device and return early if it is not. We need it to assume that it is a devicemapper for it to do the shorting of rootfs.

Reply:
So it was already a variable

https://github.com/kata-containers/runtime/blob/45fe8700b8e984fece021aac3e3bb9ddbe3c7aa3/virtcontainers/mount.go#L208